connect/basic-auth-extension/src/main/java/org/apache/kafka/connect/rest/basic/auth/extension/JaasBasicAuthFilter.java

```diff
@@ -119,16 +117,16 @@ private boolean isInternalRequest(ContainerRequestContext requestContext) {
 
     public static class BasicAuthCallBackHandler implements CallbackHandler {
 
-        private String username;
-        private String password;
+        private final String username;
```

Comment:
I have fixed all such "field not being final warnings" in https://github.com/apache/kafka/pull/15072

Please check when you get a chance.

Reply:
Thanks, I'll take a look